tools/docker/tests/test_gw_repeater.sh

```diff
@@ -76,7 +76,7 @@ main() {
         index=0
         for repeater in $REPEATER_NAMES; do
             status "Start Repeater (Remote Agent): $repeater"
-            ${scriptdir}/../run.sh ${VERBOSE_OPT} ${FORCE_OPT} start-agent -d -n ${repeater} -m aa:bb:cc:$index$index "$@"
+            ${scriptdir}/../run.sh ${VERBOSE_OPT} ${FORCE_OPT} start-agent -d -n ${repeater} -m aa:bb:cc:$index$index -- -n "$@"
```

Comment:
This leaves the Intel repeater mode vulnerable since it is not tested *at all*.
I think that a good compromise can be that the first agent is always Intel, and all the other agents are none-Intel.